lib/tasks/update_school_emails.rake

```diff
@@ -0,0 +1,8 @@
+namespace :update_school_emails do
+  desc "Update schools emails"
+  task update_emails: :environment do
+    csv_string = Rails.application.encrypted("lib/data/schools_with_emails.csv.enc", env_key: "CSV_ENCRYPTION_KEY").read
+
+    Claims::UpdateSchoolEmails.call(csv_string:)
+  end
+end
```

Comment:
This isn't `Claims` specific, should we just name this task `update_school_emails`?

`rails update_school_emails` which calls `UpdateSchoolEmails.call`.